nginx.conf

```diff
@@ -17,12 +25,45 @@ http {
     ssl_certificate /etc/letsencrypt/live/dev.telescope.cdot.systems/fullchain.pem;
     ssl_certificate_key /etc/letsencrypt/live/dev.telescope.cdot.systems/privkey.pem;
 
-    location / {
+   # Let's Encrypt route
+    location /.well-known/acme-challenge/ {
+      root /var/www/certbot;
+    }
+
+    # Non-cached routes /admin /user /health
+    location ~ ^/(admin|user|health) {
```

Comment:
Let's add `/auth` to this list too

Reply:
Added `/auth` and comments in `index.js`.